backend/migrations/versions/df361a922e41_.py

```diff
@@ -28,6 +28,10 @@ def upgrade():
     # ### commands auto generated by Alembic - please adjust! ###
     op.add_column('participant', sa.Column('new_account', sa.Boolean(), nullable=True))
     op.add_column('participant', sa.Column('use_gravatar', sa.Boolean(), nullable=True))
+    # ### end Alembic commands ###
```

Comment:
We removed this at some point because it was breaking new migrations.  Not sure why it is showing back up in this pull request, but it shouldn't be here.